src/d3a/models/strategy/storage.py

```diff
@@ -173,13 +173,13 @@ def _validate_break_even_points(break_even,
             raise ValueError("Break even point should be positive energy rate values.")
         if any(break_even_point.sell > initial_selling_rate
                for break_even_point in break_even.values()):
-            raise ValueError(f"Break even point (selling) should be lower equal "
-                             f"than initial_buying_rate")
+            raise ValueError(f"Break even point(s) (selling) should be lower equal "
```

Comment:
Very Minor: Break even point(s) (selling) should be lower OR equal

Reply:
Sorry, I dont get it. Do you want an `or` between lower and equal?
If so, shall I change it for the selling_rate error message as well?

Reply:
A ValueError is raised when break_even_point.sell is greater than initial_selling_rate. It means break_even_point.sell should either be less than or equal to initial_selling_rate.

Reply:
This is what `lower equal` means right?

Reply:
IMO, it should be "lower or equal". Im not a linguist, so may be I'm wrong.

Reply:
Ok, I just tried to google it. It is indeed not used (apart from bash scripts, where i got it probably from ) , soo, changing it to `lower or equal`